ci-scripts/build-repository.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly REPONAME=$1
+readonly PERSONAL_ACCESS_TOKEN=$2
+
+readonly APPS_FILE=./APPS
+readonly HELM_URL=https://storage.googleapis.com/kubernetes-helm
+readonly HELM_TARBALL=helm-v2.11.0-linux-amd64.tar.gz
```

Comment:
Can we make this `2.12.0` as that matches `giantswarm/helmclient`.

Reply:
Totally!